modules/static-file-service-api/src/main/java/org/opencastproject/staticfiles/endpoint/StaticFileRestService.java

```diff
@@ -204,11 +204,8 @@ public Response getStaticFile(@PathParam("uuid") String uuid) throws NotFoundExc
       final Long length = staticFileService.getContentLength(uuid);
       // It is safe to pass the InputStream without closing it, JAX-RS takes care of that
       return RestUtil.R.ok(file, getMimeType(filename), Option.some(length), Option.some(filename));
-    } catch (NotFoundException e) {
-      throw e;
-    } catch (Exception e) {
-      logger.warn("Unable to retrieve file with uuid {}", uuid, e);
-      return Response.serverError().build();
+    } catch (NotFoundException | IOException e) {
```

Comment:
I'm leery of returning a 404 on an `IOException`, but I guess without having more details of *why* we're seeing that exception this is a reasonable solution.